Makefile

```diff
@@ -1,43 +1,24 @@
-CONFIGUREFLAGS ?= --disable-tests
-SETUP           = ocaml setup.ml
 
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+.PHONY: release build install uninstall tests clean reindent
 
-doc: setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
+build:
+	jbuilder build @install --dev
 
-test: setup.data build
-	rm -f setup.data
-	make CONFIGUREFLAGS=--enable-tests build
-	$(SETUP) -test $(TESTFLAGS)
+release:
+	jbuilder build @install
 
-all:
-	$(SETUP) -all $(ALLFLAGS)
+install:
+	jbuilder install
 
-install: setup.data
-	$(SETUP) -install $(INSTALLFLAGS)
+uninstall:
+	jbuilder uninstall
 
-uninstall: setup.data
-	$(SETUP) -uninstall $(UNINSTALLFLAGS)
-	ocamlfind remove rrd || /bin/true
-
-reinstall: setup.data
-	$(SETUP) -reinstall $(REINSTALLFLAGS)
+tests:
```

Comment:
I think this is usually singular.

Reply:
I keep jumping between singular and plural :D will never get it right